security/docker/Dockerfile.node-agent-k8s

```diff
@@ -1,5 +1,4 @@
-FROM ubuntu:16.04
+FROM istionightly/base_debug
 ADD node_agent_k8s /
-RUN apt-get update && apt-get install -y ca-certificates
```

Comment:
Good to get rid of this as well - it seems it was not cleaning afterwards and was slowing down the build.